host-interaction/network/routing-table/get-routing-table.yml

```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: get routing table
+    namespace: host-interaction/network/routing-table
+    authors:
+      - michael.hunhoff@mandiant.com
+    scopes:
+      static: function
```

Comment:
```suggestion
      static: instruction
```

Reply:
Updated in https://github.com/mandiant/capa-rules/pull/1074/commits/ea84bb0da6e2b3214827fc1765dd207d407e294c